src/envVariables.js

```diff
@@ -411,6 +411,15 @@ let environmentVariables = {
 			value: 'self',
 		},
 	},
+	USER_SERVICE_TENANT_CREATION_TOPIC: {
+		message: 'Required user service tenant event',
+		optional: true,
+		requiredIf: {
```

Comment:
@priyanka-TL This is not correct, what if Kafka is ON but the adopter user service doesn't push event for tenant creation